Dangerfile

```diff
@@ -0,0 +1,57 @@
+require 'pmdtester/runner'
+require 'time'
+require 'logger'
+
+@logger = Logger.new(STDOUT)
+
+def run_pmdtester
+  Dir.chdir('..') do
+    argv = ['-r', './pmd', '-b', "#{ENV['TRAVIS_BRANCH']}", '-p', 'FETCH_HEAD', '-m', 'online']
+    Process.fork do
+	  begin
+        runner = PmdTester::Runner.new(argv)
```

Comment:
I actually don't like this interface very much, how we call the PmdTester - since it doesn't give us any benefit from just calling the command via backticks (\`pmdtester -r ./pmd ....\`).
But that's something for later, improving the API of pmdtester - it works for now.
Is this the same, what you mean with your comment "# Optimize: get more information from runner, e.g. introduce new pmd errors?"?

Reply:
Yes, this is for future expansion. e.g adding a new danger comment about whether the PR intruduce a new PMD error.